BitwardenShared/Core/Autofill/Services/AutofillCredentialService.swift

```diff
@@ -150,7 +164,16 @@ class DefaultAutofillCredentialService {
                 }
 
             if #available(iOS 17, *) {
-                let identities = decryptedCiphers.compactMap(\.credentialIdentity)
+                var identities = decryptedCiphers.compactMap(\.credentialIdentity)
+                let fido2Identities = try await clientService.platform().fido2()
+                    .authenticator(
+                        userInterface: fido2UserInterfaceHelper,
+                        credentialStore: fido2CredentialStore
+                    )
+                    .credentialsForAutofill()
+                    .compactMap { $0.toFido2CredentialIdentity() }
+
+                identities.append(contentsOf: fido2Identities)
                 try await identityStore.replaceCredentialIdentities(identities)
```

Comment:
🎨 Pretty minor, but you could combine these two arrays inline:
```suggestion
                try await identityStore.replaceCredentialIdentities(identities + fido2Identities)
```